charts/civil-ga-ccd/values.preview.template.yaml

```diff
@@ -206,7 +206,7 @@ ccd:
         IDAM_SERVICE_KEY: ${CCD_API_GATEWAY_S2S_SECRET}
         IDAM_OAUTH2_CLIENT_SECRET: ${CCD_API_GATEWAY_IDAM_CLIENT_SECRET}
         PROXY_DOCUMENT_MANAGEMENT: http://dm-store-aat.service.core-compute-aat.internal
-      ingressHost: gateway-${SERVICE_FQDN}
+      ingressHost: gateway-${SERVICE_NAME}.preview.platform.hmcts.net
```

Comment:
on review all these changes are done wrongly . .so will raise another PR closing this